includes/base_controls/QControlProxy.class.php

```diff
@@ -62,7 +60,7 @@ public function RenderAsHref($strActionParameter = null, $blnDisplayOutput = tru
 			else
 				$this->strTargetControlId = $this->objForm->GenerateControlId();
 			
-			$this->strActionParameter = $strActionParameter;
+			$this->mixActionParameter = $strActionParameter;
```

Comment:
Just make a new directory under your docroot and follow the alpha3 installation instructions. Composer does its work with git, so there shouldn't be any additional git commands to issue. The only additional thing at the end would be to install the examples database.